### ISSUE TITLE
Add version info to exe files

### ISSUE THREAD

**Issue by yfdyh000** [getlantern/lantern#3606]
The lantern.exe and lantern-installer.exe files no info in Property - Details, can't to distinguish it without run it.

**Comment by myleshorton** [comment]
Yup good point @yfdyh000 -- noted thanks. The installer creation file is here if you want to try a patch:

https://github.com/getlantern/lantern/blob/valencia/installer-resources/windows/lantern.nsi

lantern.exe would probably take a little more investigating.
